docs/faq.md

```diff
@@ -35,4 +38,12 @@ helm install azure/open-service-broker-azure --name osba --namespace osba \
   --set azure.clientId=$AZURE_CLIENT_ID \
   --set azure.clientSecret=$AZURE_CLIENT_SECRET \
   --set modules.minStability=EXPERIMENTAL
+```
+
+## "osba" is forbidden: not yet ready to handle request
+
+[Service Catalog](https://github.com/kubernetes-incubator/service-catalog) in used to integrate Open Service Broker for Azure. Service Catalog currently works with Kubernetes version 1.9.x and higher. If you are receive this error message when trying to install Open Service Broker For Azure, check the version of your Kubernetes cluster. If you are running a version less than 1.9, please upgrade. If you are making a new cluster with AKS, you can specify the version with the `--kubernetes-version` flag like so:
```

Comment:
s/in used/is used/

> integrate Open Service Broker for Azure

Qualify this:

> integrate Open Service Broker for Azure with Kubernetes clusters.

Per conversation in slack: 

> Kubernetes version 1.9.x and higher

s/1.9.x/1.9.0/

> If you are receive this error message

s/are//